src/style.css

```
@@ -1,3 +1,72 @@
-body {
-  margin: 0;
-}
+html {
+  font-family: Roboto, sans-serif;
+       }
+
+    body {
+      margin: 0;
+       }
+
+       .header {
+       align-items: center;
+       justify-content: space-between;
+       margin: 0;
+       padding: 0 50px;
+       display: flex;
+       min-width: 1024px;
+       }
+
+       .a {
+        text-decoration: none;
+       }
+
+       img {
+        width: 40px;
+        height: 40px;
+
+         }
```

```suggestion
     img {
       width: 40px;
       height: 40px;
      }
```
It is better to use css classes for styling purposes